esrally/resources/logging.json

```diff
@@ -20,14 +20,14 @@
   "handlers": {
     "rally_log_handler": {
       "class": "logging.handlers.WatchedFileHandler",
-      "filename": "${LOG_PATH}/rally.log",
+      "filename": "${LOG_PATH}rally.log",
```

Comment:
Ugh it's unfortunate it has to look like that, but as we are currently consistently using the `os` module I don't see a workaround.